ooni/agent/agent.py

```diff
@@ -6,9 +6,21 @@
 from ooni.agent.scheduler import SchedulerService
 
 class AgentService(service.MultiService):
+    """Contain any element to manage agent services.
+
+
+    The class above inherits to another class wich is a services container."""
+
+
     def __init__(self, web_ui_port):
-        service.MultiService.__init__(self)
+        """Load configuration or made the default configuration of the service.
 
+
+        Set up the child services of the AgentService.
+        If the advanced configuration is not enabled, the page is not displayed.
```

Comment:
You should specify what is meant by "advanced configuration". You probably want to say:

"If `config.advanced.disabled_webui` is set to True, then the web UI will not be started"